recipe/meta.yaml

```diff
@@ -23,9 +23,12 @@ source:
     # CMake finds iconv_open in glibc, but finds iconv.h from libiconv.
     # Make sure both are found from one place.
     - find_iconv.patch                                                           # [not win]
+    # Doxygen 1.8.13 sometimes segfaults when building the XML output:
+    # https://github.com/doxygen/doxygen/pull/555
+    - doxygen-1.8.13-NULL-dereference.patch
```

Comment:
Good catch! If someone gets this problem on Windows, this recipe will have to be changed to build on source for Windows too.